model/transformations/predicitions_to_text.py

```diff
@@ -0,0 +1,28 @@
+import pandas as pd
+from .base import SimpleTransformation
+from configs.constants import Const
+
+
+class PredictionsToText(SimpleTransformation):
+    def predict(self, dataset: pd.DataFrame) -> pd.DataFrame:
+
+        agg_text = map(
+            str,
+            zip(
+                map(str, dataset[Const.preds_col]),
+                map(str, dataset[Const.probs_col]),
+            ),
+        )
+        if Const.label_col in dataset.columns:
```

Comment:
this is very inconvenient, and I think we should probably treat labels separately, so we don't need to retain it everywhere where we concatenate, etc.